README.md

```diff
@@ -38,8 +38,8 @@ $ sbt
 > exit
 $ cd target/sbt-test/default-*/scripted
 $ sbt
-$ jetty:start
-$ browse # starts browser for you, or manually open http://localhost:8080 to verify
+> jetty:start
+-> executes browser, and opens `http://localhost:8080` to verify
```

Comment:
When trying according to the procedure, it seems that at the time of jetty:start, the sbt command line is occupied and there is no timing to start the 'browse' command.

Another sbt is launched and the browse command displays the following error.
```
Not a valid command: browse (similar: set)
```